src/app/work-item/work-item.service.ts

```diff
@@ -198,8 +199,11 @@ export class WorkItemService {
     if (this.prevFilters.length) {
       for (let i = 0; i < this.prevFilters.length; i++) {
         // In case of assignee filter
-        if (this.prevFilters[i].id === 1 && this.prevFilters[i].active === true) {
-          if (workItem.relationalData.assignees.findIndex(item => item.id == this.prevFilters[i].value) === -1) {
+        if (this.prevFilters[i].id === 1
+            && this.prevFilters[i].active === true) {
```

Comment:
Did you mean !!this.prevFilters[i].id === true === !!this.prevFilters[i].active ?